src/app/cards/services/cards-store.service.ts

```diff
@@ -6,21 +6,13 @@ import * as fromStore from '../store';
   providedIn: 'root'
 })
 export class CardsStoreService {
-  public searchTerm$ = this.store.select(fromStore.selectSearchTerm);
   public cards$ = this.store.select(fromStore.selectCards);
+  public selectedCard$ = this.store.select(fromStore.selectedCard);
   public cardsLoading$ = this.store.select(fromStore.selectCardsLoading);
   public cardsLoaded$ = this.store.select(fromStore.selectCardsLoaded);
   public cardErrors$ = this.store.select(fromStore.selectCardsError);
 
   constructor(
     private store: Store<fromStore.CardsState>
   ) { }
-
-  public updateSearch(search: string): void {
-    this.store.dispatch(fromStore.updateSearch({ search }));
-  }
-
-  public getCards(searchTerm: string, page: string = '1', perPage?: string): void {
-    this.store.dispatch(fromStore.loadCards({ search: searchTerm, page, perPage }));
-  }
```

Comment:
Don't need these anymore since we're handling search changes from the query params.